### PR TITLE
Pagination attempt

### PR DESCRIPTION
**Pull request by seaninryan**
Hey @tombee 

Fire this up and see what you think. It makes pagination *work* on the `#/images` page by adding some css & the relevant parameters to the reactable `<Table>` element

It's a bit jumpy... maybe we can do better

### CODE REVIEW

File: ui/src/css/reactable.css [hunk 1]
```diff
@@ -5,3 +5,35 @@ th.reactable-header-sort-asc:after {
...
+    border-top-right-radius: 4px;
+    border-bottom-right-radius: 4px;
+    border-right-width: 1px;
+}
+
+tbody.reactable-pagination tr td > a {
+    position: relative;
+    padding: 6px 12px;
+    line-height: 1.42857143;
+    color: $blue;
```
Comment (tombee) [inline ui/src/css/reactable.css:34]: We don't have any css pre-processing, so I guess this probably needs to be a hex colour value.

**Comment by tombee** [comment]
Hmm.. yeah.. it's a bit weird how it sticks the pagination controls in another `tbody` instead of a footer or something.  I see what you mean with it being "jumpy", could probably do with `Previous` and `Next` replacing with arrows, and disabling them when they're not applicable.

2017-01-31-20-32-56_1220x616

**Comment by seaninryan** [comment]
Yeah disabled would be better. Also when the are >`pageButtonLimit` pages is not clear why. Needs to have some `...`s in there or something

**Comment by tombee** [comment]
@seaninryan should we close this in favour of #957?  That one looks a load better than what we have already.

**Comment by seaninryan** [comment]
Agreed @tombee, closing

**Pull request closed without merging**